tests/com/tony/server/WorkerThreadTest.java

```diff
@@ -1,37 +1,98 @@
 package com.tony.server;
 
+import com.tony.server.response.HeadResponse;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 import java.io.*;
```

Comment:
Don't use wildcard imports, it's better to be explicit with what you are bringing in.

If you don't like the longer list of imports at the top: http://vim.wikia.com/wiki/Folding